core/src/main/kotlin/com/strumenta/kolasu/model/Printing.kt

```diff
@@ -18,9 +18,8 @@ fun Node.relevantMemberProperties(withPosition: Boolean = false, withNodeType: B
             !it.name.startsWith("component") &&
                 (it.name != "position" || withPosition) &&
                 (it.name != "nodeType" || withNodeType) &&
-                it.name != "specifiedPosition" &&
                 it.name != "properties" &&
-                it.name != "parseTreeNode" &&
+                it.name != "origin" &&
```

Comment:
Actually, it looks like DEFAULT_IGNORED_PROPERTIES is redundant and unnecessary since _nodeProperties_ are already filtered. But we cannot use _nodeProperies_ here because it excludes the position.

Reply:
And, actually, some properties are already filtered as _@Derived_, so probably another improvement would be to get rid of these lists of properties and just use _@Derived_ everywhere

Reply:
I would not use @Derived but probably creating a separate annotation as the meaning is a bit different. Derived here means "calculated from other values"

Reply:
Ok, I thought of `@Derived` b/c _Node_ already has its _properties_ property marked as `@Derived`, but we could introduce another annotation, say `@BuiltIn`

Reply:
I've settled on `@Internal` for this.